VULNERABILITY.md

```diff
@@ -0,0 +1,7 @@
+Vulnerability Disclosure notice needs to be included within the repo
+
+**Message:
+Do not file an issue on Github for security issues. Please review the guidelines for disclosure. Reports should be encrypted using PGP (public key) and sent to security@linkedin.com preferably with the title "Vulnerability in Github LinkedIn/REPO - <short summary>".
```

Comment:
```suggestion
Please do not file an issue on Github for security issues. Please review the guidelines for disclosure. Reports should be encrypted using PGP (public key) and sent to security@linkedin.com preferably with the title "Vulnerability in Github LinkedIn/REPO - <short summary>".
```

Reply:
you may want to link to the LI public PGP key directly https://www.linkedin.com/help/linkedin/answer/79676